### ISSUE TITLE
Feature Request - _RunFSIScript

### ISSUE THREAD

**Issue by EricM81** [goswinr/Fesh.Rhino#4]
I'd love to have an FSI implementation of `_RunPythonScript`.   I work with non-developers and need a clean way to give them scripts without them needing to look at a code editor.  They are familiar with executing Python scripts, so following that UX would have the least friction.

Could be used from a button with the `-` operator, i.e., `-RunFSIScript C:\Users\EricM\Source\Repos\RhinoFSI\doTheThing.fsx`
Could be called interactively as `_RunFSIScript`, where it pops up a file open dialog to select and run a file.

**Comment by goswinr** [comment]
Yes I want that feature too. Maybe compiling the script first to save the few seconds startup time too?

**Comment by goswinr** [comment]
Compiling scripts for faster loading is already possible

**Comment by goswinr** [comment]
> I work with non-developers and need a clean way to give them scripts without them needing to look at a code editor.

There are several ways to achieve that. 

Would that be over a trusted local network or sharepoint? Do their PCs have the dotnet SDK installed? Are you willing to install and update Fesh.Rhino on those PCs? 

Already today you could create a python command script that just loads the f# DLL created with the current "compile script" command in Fesh. This would have the fastest startup time and not require any installation on your colleagues PCs

**Comment by EricM81** [comment]
I have no problem installing the Fesh plugin on each machine.  Plus, I'd like to avoid the loading delay the first time you run a python script.

We sync between PCs with Resilio Sync.  It's like rsync on Linux.  The scripts, toolbars, plugins, and template geometry are pushed to a read-only on each machine's `J:\` drive.   When the master machine updates anything, even plugins, the rest of the team implicitly gets updated.
